src/components/common/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -259,10 +256,6 @@ export default class PeopleAndRoles extends Mixins(CommonMixin, DateMixin, OrgPe
   get originalParties (): OrgPersonIF[] {
     if (this.getOriginalIA) {
       return this.getOriginalIA.parties
-    } else if (this.getOriginalChgRegistration) {
-      return this.getOriginalChgRegistration.parties
-    } else if (this.getOriginalRegistration) {
-      return this.getOriginalRegistration.parties
```

Comment:
Firm corrections don't load the original Registration / ChangeOfRegistration. Instead, firm corrections load a business snapshot. So, the original else case below works just fine.